js/README.md

```diff
@@ -145,39 +145,53 @@ This project is a library for running ONNX models on browsers. It is the success
    2. in `<ORT_ROOT>/js/common/`, run `npm ci`.
    3. in `<ORT_ROOT>/js/web/`, run `npm ci`.
 
-2. ~~Follow [instructions](https://www.onnxruntime.ai/docs/how-to/build.html#apis-and-language-bindings) for building ONNX Runtime WebAssembly. (TODO: document is not ready. we are working on it. Please see steps described as below.)~~
+2. Prepare ONNX Runtime WebAssembly artifacts.
 
-   in `<ORT_ROOT>/`, run one of the following commands to build WebAssembly:
+   You can either use the prebuilt artifacts or build it by yourself.
 
-   ```sh
-   # In windows, use 'build' to replace './build.sh'
+   - Setup by script.
 
-   # The following command build debug.
-   ./build.sh --build_wasm
+     In `<ORT_ROOT>/js/web/`, run `npm run pull:wasm` to pull WebAssembly artifacts for latest master branch from CI pipeline.
 
-   # The following command build debug with debug info.
-   ./build.sh --build_wasm --skip_tests --enable_wasm_debug_info
+   - Download artifacts from pipeline manually.
 
-   # The following command build release.
-   ./build.sh --config Release --build_wasm --skip_tests --disable_wasm_exception_catching --disable_rtti
-   ```
+     you can download prebuilt WebAssembly artifacts from [Windows WebAssembly CI Pipeline](https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=161&_a=summary). Select a build, download artifacts "Release_ort-wasm" and "Release_ort-wasm-threaded" and unzip. See instructions below to put files into destination folders.
+
+   - Build WebAssembly artifacts.
+
+     1. Build ONNX Runtime WebAssembly
+
+        ~~Follow [instructions](https://www.onnxruntime.ai/docs/how-to/build.html#apis-and-language-bindings) for building ONNX Runtime WebAssembly. (TODO: document is not ready. we are working on it. Please see steps described as below.)~~
+
+        in `<ORT_ROOT>/`, run one of the following commands to build WebAssembly:
+
+        ```sh
+        # In windows, use 'build' to replace './build.sh'
+
+        # The following command build debug.
+        ./build.sh --build_wasm
+
+        # The following command build debug with debug info.
+        ./build.sh --build_wasm --skip_tests --enable_wasm_debug_info
 
-   To build with multi-thread support, append flag `--enable_wasm_threads` to the command. Make sure to build both single-thread and multi-thread before next step.
+        # The following command build release.
+        ./build.sh --config Release --build_wasm --skip_tests     --disable_wasm_exception_catching --disable_rtti
```

Comment:
remove some spaces between '--skip_tests' and '--disable_wasm_exception_catching'